src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -260,6 +260,9 @@
     <Compile Include="$(CommonPath)\Interop\Unix\System.Security.Cryptography.Native\Interop.Crypto.cs">
       <Link>Common\Interop\Unix\System.Security.Cryptography.Native\Interop.Crypto.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\System.Security.Cryptography.Native\Interop.X509Name.cs">
```

Comment:
This confuses me.  You've not made any changes that I see that would require adding this file to X509Certificates.